docs/source/markdown/options/latest.md

```diff
@@ -0,0 +1,10 @@
+####> This option file is used in:
+####>   podman attach, container inspect, diff, exec, init, inspect, kill, logs, mount, network reload, pause, pod inspect, pod kill, pod logs, pod rm, pod start, pod stats, pod stop, pod top, port, restart, rm, start, stats, stop, top, unmount, unpause, wait
+####> If you edit this file, make sure your changes
```

Comment:
New use of "you" -- my fault, sorrynotsorry

Reply:
The lines starting with `####>` are all auto-generated now, correct?

Reply:
Yes, and not user-visible.